mordant/build.gradle.kts

```diff
@@ -44,9 +55,10 @@ kotlin {
         val mingwX64Main by getting { dependsOn(nativeMain) }
         val posixMain by creating { dependsOn(nativeMain) }
         val linuxX64Main by getting { dependsOn(posixMain) }
-        val macosMain by creating { dependsOn(posixMain) }
-        listOf("macosX64", "macosArm64").forEach { target ->
-            getByName(target + "Main").dependsOn(macosMain)
+        val darwinMain by creating { dependsOn(posixMain) }
+        darwinTargets.forEach {
+            getByName("${it.targetName}Main").dependsOn(darwinMain)
+            getByName("${it.targetName}Test").dependsOn(commonTest)
```

Comment:
Is `dependsOn(commonTest)` necessary? I thought commonMain and commonTest were implicit.